### PR TITLE
Jupyterhub update dependencies

### PR DESCRIPTION
**Pull request by annoyingllama**
I confirm that I have added:

- [x] A comment to every Dockerfile with information about target repository and tag (version).
- [x] Filled in readme on dockerhub and in this repo about how to build the image if some extra steps should be done.

### CODE REVIEW

**Review by mpasa — APPROVED** [review]
If you deploy this, can we also put a variable pointing to the DS credentials for Google on the image used by drivers?

**Comment by annoyingllama** [comment]
> If you deploy this, can we also put a variable pointing to the DS credentials for Google on the image used by drivers?

what do you mean? maybe use this branch

File: jupyterhub/data-science-notebook/Dockerfile [hunk 1]
```diff
@@ -21,11 +21,17 @@ RUN ./almond_211 --install --jupyter-path /usr/local/share/jupyter/kernels --id
...
 RUN ./coursier bootstrap \
     -r jitpack \
-    -i user -I user:sh.almond:scala-kernel-api_2.12.11:0.10.0 \
-    sh.almond:scala-kernel_2.12.11:0.10.0 \
+    -i user -I user:sh.almond:scala-kernel-api_2.12.11:0.10.9 \
+    sh.almond:scala-kernel_2.12.11:0.10.9 \
     -o almond_212
 
-RUN ./almond_212 --install --jupyter-path /usr/local/share/jupyter/kernels --id scala_2_12_11  --display-name "Scala 2.12.11"
+RUN ./almond_212 --install --jupyter-path /usr/local/share/jupyter/kernels --id scala_2_12_11  --display-name "Scala 2.12.11" --env GOOGLE_APPLICATION_CREDENTIALS=/home/jovyan/shared/credentials/ds-credentials.json
```
Comment (annoyingllama) [inline jupyterhub/data-science-notebook/Dockerfile:28]: LGTM

**Comment by annoyingllama** [comment]
released image `pubnative/jupyterhub:spark3.2-hadoop3.2-py3.9-java8-8`

**Pull request merged**